src/gui/editors/AutomationEditor.cpp

```diff
@@ -1105,7 +1110,11 @@ inline void AutomationEditor::drawCross( QPainter & p )
 		mouse_pos.y() >= 0 &&
 		mouse_pos.y() <= height() - SCROLLBAR_SIZE )
 	{
-		QToolTip::showText( tt_pos, QString::number( scaledLevel ), this );
+		if (m_pointYLevel == 0) {
+			QToolTip::showText( tt_pos, QString::number( scaledLevel ), this );
+		} else if (m_pointYLevel != 0) {
+			QToolTip::showText( tt_pos, QString::number( m_pointYLevel ), this );
+		}
```

Comment:
I wasn't sure about that... simply because the wiki also states:
"The majority of LMMS conventions follows the Qt Coding Style"
and according to Qt Coding:
"Use attached braces: The opening brace goes on the same line as the start of the statement. If the closing brace is followed by another keyword, it goes into the same line as well"

`// Wrong
 if (codec)
 {
 }
 else
 {
 }

 // Correct
 if (codec) {
 } else {
 }`

I had interpreted the LMMS coding conventions to suggest that you should use brackets as opposed to non-explicit blocking:

`// BAD - Example of not explicit blocking
if (m_sample > 0)
	--m_sample;`

Seems like a contradiction of conventions, perhaps some clarification in the wiki would be ideal.

Reply:
I suppose the combined use of the words "explicit blocking" and "Block brackets SHOULD be on their own line" (I misinterpreted) is clear enough to suggest that brackets should not start on the beginning line of the statement.  With the exception of the whole statement fitting on one line, of course.